spec/schemas/_common.yaml

```diff
@@ -20,137 +22,164 @@ components:
       required:
         - acknowledged
     DocStatus:
-      description: |-
-        The item level REST category class codes during indexing.
+      description: The item level REST category class codes during indexing.
```

Comment:
```suggestion
      description: The item level HTTP response status code during indexing.
```

@nhtruong: Per your comment, do you think this description is better?

Reply:
Yes. That's more accurate :)